packages/core/docs/guide/cart-and-wishlist.md

```diff
@@ -253,170 +232,130 @@ The `load` method will load your cart from the server or create a new one if it
+
+export default {
+  setup() {
+    const { addItem } = useWishlist();
+    const { result } = useFacet();
+    const products = computed(() => facetGetters.getProducts(result.value));
+
+    return {
+      products,
+      addItem,
+      productGetters
+    };
   }
+};
 </script>
 ```
 
 ## Removing an item from the wishlist
- 
+
 To remove an item from the cart use `removeItem` method.
 
 ```vue
 <template>
-  // ...
-    <div>
-      <ul>
-        <li
-          v-for="product in products" :key="product.id"
-        >
-          ...
-          <button
-            @click="removeItem({ product })"
-          >
-            Clear wishlist
-          </button>
-        </li>
-      </ul>
-      <span>
-        {{ totals.total }}
-      </span>
-      <span>
-        {{ totalItems }}
-      </span>        
-    </div>
-  // ...
-</template>   
+  <!-- ... -->
+  <div>
+    <ul>
+      <li v-for="item in wishlistItems" :key="wishlistGetters.getWishlistItemSku(product)">
+        <!-- ... -->
+        <button @click="removeItem({ item })">
+          Remove item
+        </button>
+      </li>
+    </ul>
+    <span>
+      {{ totalItems }}
+    </span>
+  </div>
+  <!-- ... -->
+</template>
 <script>
-  import { computed } from '@vue/composition-api';
-  import { useWishlist, wishlistGetters } from '{INTEGRATION}';
-  import { onSSR } from '@vue-storefront/core';
-  export default {
-    setup() {
-      const {
-        wishlist, 
-        removeItem, 
-        loading
-      } = useWishlist();
-    
-    const products = computed(() => wishlistGetters.getItems(wishlist.value));
-    const totals = computed(() => wishlistGetters.getTotals(wishlist.value));
-    const totalItems = computed(() => wishlistGetters.getTotalItems(wishlist.value));
-
-      return {
-        products,
-        totals,
-        totalItems,
-        removeItem,
-        loading
-      };
-    }
-  };
+import { computed } from '@vue/composition-api';
+import { useWishlist, wishlistGetters } from '{INTEGRATION}';
+import { onSSR } from '@vue-storefront/core';
+export default {
+  setup() {
+    const { wishlist, removeItem } = useWishlist();
+
+    const wishlistItems = computed(() => wishlistGetters.getItems(wishlist.value));
+    const totalItems = computed(() =>
+      wishlistGetters.getTotalItems(wishlist.value)
+    );
+
+    // load wishlist if it's not loaded before
+
+    return {
+      wishlistItems,
+      totalItems,
+      removeItem
+    };
+  }
+};
 </script>
 ```
 
-## Checking if an item is on the wishlist 
+## Checking if a single item is on the wishlist
 
 To check if a product is already on the wishlist pass it to `isInWishlist` method:
 
 ```vue
-<template>
-  // ...
-    <ul>
-      <li
-        v-for="product in products" :key="product.id"
-      >
-        <div
-          :isAddedToWishlist="isInWishlist({ product })"
-        >
-        </div>
-      </li>
-    </ul>
-  // ...
-</template>    
```

Comment:
Is there a reason this was removed? I think it fits this example :)

Reply:
I'm pretty sure it was discussed before and as the result should be removed, because it's not necessary. But I couldn't find this conversation. So I restored it and did accordingly in the "Checking if an item is in the cart" chapter.